### PR TITLE
Add article about rails conf workshop

### PR DESCRIPTION
**Pull request by lubc**
@etagwerker This PR adds a new article to talk about your Rails Upgrade workshop for the Rails Conf. 

Let me know if this is what you had in mind. Thanks!

### CODE REVIEW

File: _posts/2019-04-26-workshop-rails-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "Workshop - Upgrade Rails 101: The Roadmap to Smooth Upgrades"
+date: 2019-04-05 10:00:00
```
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: Fix the date

File: _posts/2019-04-26-workshop-rails-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+categories: ["upgrade-rails", "workshop"]
+author: "luciano"
+---
+
+Next week the [Rails](https://rubyonrails.org/) community is having the biggest event of the year, the [Rails Conf 2019](https://railsconf.com/). We participated on previous editions of this conference but this time we have the honor to be in the speakers side of it. We'll be teaching you what we're best at: [Rails Upgrades](https://fastruby.io/blog/tags/upgrades).
```
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: "We participated IN previous editions of this conference but this time we have the honor to be ON the speakers side of it."

File: _posts/2019-04-26-workshop-rails-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+<!--more-->
+
+<img src="/blog/assets/images/workshop-rails-conf.png" alt="Workshop Rails Conf">
+
+At [Fast Ruby](https://fastruby.io/) we have been doing Rails Upgrades for many years. That allowed us to work on all kind of projects. From tiny [MVP's](https://fastruby.io/minimum-viable-product) to huge [monolith](https://fastruby.io/monolith) applications.
```
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: "THIS HAS allowed us to work..."

File: _posts/2019-04-26-workshop-rails-upgrade.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+During the 2-hour [workshop](https://railsconf.com/program/workshops#session-776) you will learn a proven Rails upgrade process for major and minor version changes of Rails.
+We will teach you how you can [evaluate](https://fastruby.io/blog/upgrade-rails/legacy-rails-silently-judging-you.html) a Rails application before consider upgrading it. What tools you should us to make the upgrade easier. What are the main things that you should address on every upgrade. And much more.
```
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: "before considerING upgrading it. What tools you should usE to make the upgrade easier. What THE MAIN THINGS ARE that you should address one every upgrade."

File: _posts/2019-04-26-workshop-rails-upgrade.md (hunk 2 above)
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: Just "Rails Conf", not "the Rails Conf"

File: _posts/2019-04-26-workshop-rails-upgrade.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+You will leave this workshop with a roadmap to upgrade your Rails application.
+
+If you're going to the Rails Conf and you are interested on attending our workshop, you can book your place [here](https://www.eventbrite.com/e/workshop-upgrade-rails-101-the-roadmap-to-smooth-upgrades-tickets-60333391799). In that link there are a few notes for you to be prepared before the workshop. So make sure to check that out.
```
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: Just "Rails Conf" not "the Rails Conf"
"and ARE interested IN attending"
"In that link there are a few notes TO HELP YOU be prepared FOR the workshop. Make sure to check IT out."

File: _posts/2019-04-26-workshop-rails-upgrade.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+Hope we can see you there!
```
Comment (ercohen14) [inline _posts/2019-04-26-workshop-rails-upgrade.md]: "Hope TO see you there!"

**Comment by lubc** [comment]
@ercohen14 Thanks for your feedback 👍 Just addressed it

**Pull request merged**